src/settings_translation_file.cpp

```diff
@@ -286,32 +286,32 @@ fake_function() {
 	gettext("Light curve mid boost center");
 	gettext("Center of light curve mid-boost.");
 	gettext("Light curve mid boost spread");
-	gettext("Spread of light curve mid-boost.\nStandard deviation of the mid-boost gaussian.");
+	gettext("Spread of light curve mid-boost.\nStandard deviation of the mid-boost Gaussian.");
 	gettext("Texture path");
 	gettext("Path to texture directory. All textures are first searched from here.");
 	gettext("Video driver");
-	gettext("The rendering back-end for Irrlicht.\nA restart is required after changing this.\nNote: on Android, stick with OGLES1 if unsure! App may fail to start otherwise.\nOn other platforms, OpenGL is recommended, and it’s the only driver with\nshader support currently.");
+	gettext("The rendering back-end for Irrlicht.\nA restart is required after changing this.\nNote: On Android, stick with OGLES1 if unsure! App may fail to start otherwise.\nOn other platforms, OpenGL is recommended, and it’s currently the only driver with\NShader support.");
```

Comment:
`\nshader` is a line break followed by the word "shader", it does not refer to NShader.